tests/index/index_death_tests.cpp

```diff
@@ -291,13 +291,9 @@ TEST(index_death_test_formats_10, index_meta_write_fail_1st_phase) {
   }
 
   {
-    const auto all_features = irs::flags{
-      irs::type<irs::document>::get(),
-      irs::type<irs::frequency>::get(),
-      irs::type<irs::position>::get(),
-      irs::type<irs::payload>::get(),
-      irs::type<irs::offset>::get()
-    };
+    constexpr irs::IndexFeatures all_features =
```

Comment:
We cold benefit here  from having ALL_FEATURES value defined in IndexFeatures :)